docs/charter.md

```diff
@@ -1,7 +1,11 @@
 ICU4X Charter
 ==============
 
-ICU4X is a new project whose objective is to solve the needs of clients who wish to provide client-side i18n for their products in resource-constrained environments.
+ICU4X is a project whose objective is to solve the needs of clients who wish to provide client-side i18n for their products in resource-constrained environments.
```

Comment:
"objective to solve the needs…" is perhaps too many levels of indirection. How about something like, "The ICU4X project provides client-side i18n for resource-constrainied environments"?